tests/clone_test.go

```diff
@@ -0,0 +1,53 @@
+package tests_test
+
+import (
+	"flag"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	k8sv1 "k8s.io/api/core/v1"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/containerized-data-importer/tests"
+)
+
+const (
+	TestNS = "default"
+)
+
+var _ = Describe("Cloning", func() {
+	var (
+		err error
+	)
+
+	flag.Parse()
```

Comment:
This relies on _utils.go_'s `init` func. Maybe this should be documented/commented somewhere? I've been thinking that it might be better to move `init` to a flags helper func and have it set a pkg global flags struct so that all tests have easier access to all flag values.